SU-HSDO/suhumsci
# Shs 97 photo album raised cards

## Summary
Adds the `box-shadow` effect to the photo album grid when the appropriate class is added to the layout.

## Steps to Test
1. Either create a photo album or visit the preexisting one (`/photo-album`).
2. Edit the layout and in the 'Main Region' area, add the `hb-raised-cards` class to Field Formatter class area.
3. Revisit the page and note that the Photo Album slider should be unaffected.
4. The Photo Album grid should have the box shadow effect. See screenshots below, but Colorful has a new `box-shadow` effect while Traditional repurposes a preexisting mixin along with `padding` around the image.

## Screenshots
Colorful:
![Screen Shot 2022-11-02 at 11 02 39 AM](https://user-images.githubusercontent.com/8405274/199524960-b6ecbfad-6d0f-41d5-a82d-0952560136c1.png)

Traditional:
 
![raisedcards](https://user-images.githubusercontent.com/8405274/202004637-3151e462-f546-47a5-a283-fb8acfbfc645.png)
